### ISSUE TITLE
Embedding Projector standalone mode: "target 'devserver' not declared in package "

### ISSUE THREAD

**Issue by luggie** [tensorflow/tensorboard#4153]
Getting these error messages when trying to run bazel as stated in README.md

 ```
ERROR: Skipping 'tensorboard/plugins/projector/vz_projector:devserver': no such target '//tensorboard/plugins/projector/vz_projector:devserver': target 'devserver' not declared in package 'tensorboard/plugins/projector/vz_projector' defined by /mnt/5964E79723485700/GoogleDrive/2-UNI/faecher/MA/tensorboard/tensorboard/tensorboard/plugins/projector/vz_projector/BUILD
WARNING: Target pattern parsing failed.
ERROR: no such target '//tensorboard/plugins/projector/vz_projector:devserver': target 'devserver' not declared in package 'tensorboard/plugins/projector/vz_projector' defined by /mnt/5964E79723485700/GoogleDrive/2-UNI/faecher/MA/tensorboard/tensorboard/tensorboard/plugins/projector/vz_projector/BUILD
INFO: Elapsed time: 0.086s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (0 packages loaded)
FAILED: Build did NOT complete successfully (0 packages loaded)
```

Running on Ubuntu 18.04. I tried installing bazel with binary installer and apt's repository. Both with the same result. Bazel is correctly installed (3.5.0)

**Comment by aknoerig** [comment]
This is probably due to an outdated readme documentation. The build target was apparently renamed to `standalone`.
The build command should now be:

`bazel run tensorboard/plugins/projector/vz_projector:standalone`

**Comment by aknoerig** [comment]
I just filed a pull-request to fix this: #4155

**Comment by stephanwlee** [comment]
Fixed with #4155.
